nmap/public/manifest.json

```diff
@@ -0,0 +1,26 @@
+{
+  "title": "nmap",
+  "name": "nmap",
+  "description": "Perform nmap scans on a client!",
+  "icon": "icon.png",
+  "author": {
+    "name": "m5kro",
+    "email": "m5kro@proton.me"
+  },
+  "keywords": [
+    "nmap"
+  ],
+  "repository": "https://github.com/m5kro/frieren-module-nmap",
```

Comment:
As it will be integrated to this repo the repository and bugs values should be pointing here. 
Ideally we should do something similar to what HomeAsistant does and mark the owners of the modules and so when there is a bug/prs/etc it auto tags the responsible ones.
 I never understood if they do it manually or it is a hook (I think it is the latter).